src/agents/constants.ts

```diff
@@ -1,37 +1,58 @@
 import { ChecksumAddress } from '../types';
 
-interface Contracts {
-  NU: ChecksumAddress;
-  STAKINGESCROW: ChecksumAddress;
-  POLICYMANAGER: ChecksumAddress;
-  ADJUDICATOR: ChecksumAddress;
+export enum Contract {
+  STAKING_ESCROW = 'STAKING_ESCROW',
+  SUBSCRIPTION_MANAGER = 'SUBSCRIPTION_MANAGER',
 }
 
-const MAINNET: Contracts = {
-  NU: '0x4fE83213D56308330EC302a8BD641f1d0113A4Cc',
-  STAKINGESCROW: '0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2',
-  POLICYMANAGER: '0x67E4A942c067Ff25cE7705B69C318cA2Dfa54D64',
-  ADJUDICATOR: '0x359924Be0640659F34198e518BF3d40Fb56160BE',
+type ContractAddress = {
+  STAKING_ESCROW: ChecksumAddress;
+  SUBSCRIPTION_MANAGER: ChecksumAddress;
 };
 
-const GOERLI: Contracts = {
-  NU: '0x02B50E38E5872068F325B1A7ca94D90ce2bfff63',
-  STAKINGESCROW: '0x40Ca356d8180Ddc21C82263F9EbCeaAc6Cad7250',
-  POLICYMANAGER: '0xaC5e34d3FD41809873968c349d1194D23045b9D2',
-  ADJUDICATOR: '0xC62e20B599416e4B5F3b54d50837F070bFec6412',
+const MAINNET: Partial<ContractAddress> = {
+  STAKING_ESCROW: '0xbbD3C0C794F40c4f993B03F65343aCC6fcfCb2e2',
 };
 
-const RINKEBY: Contracts = {
-  NU: '0x78D591D90a4a768B9D2790deA465D472b6Fe0f18',
-  STAKINGESCROW: '0x6A6F917a3FF3d33d26BB4743140F205486cD6B4B',
-  POLICYMANAGER: '0x4db603E42E6798Ac534853AA2c0fF5618cb0ebE5',
-  ADJUDICATOR: '0xE1d0C09b94ba522BCC1b73922dc1f0b6ca9bEA26',
+const GOERLI: Partial<ContractAddress> = {
+  STAKING_ESCROW: '0x40Ca356d8180Ddc21C82263F9EbCeaAc6Cad7250',
 };
 
-export const CONTRACTS: Record<ChecksumAddress, Contracts> = {
-  mainnet: MAINNET,
-  goerli: GOERLI,
-  rinkeby: RINKEBY,
+const RINKEBY: Partial<ContractAddress> = {
+  STAKING_ESCROW: '0x6A6F917a3FF3d33d26BB4743140F205486cD6B4B',
+};
+
+const MUMBAI: Partial<ContractAddress> = {
+  SUBSCRIPTION_MANAGER: '0x5029dA19035f24b04229dBc6e73463b015F5E071',
```

Comment:
What address is this?  The `SubscriptionManager` is deployed to Mumbai here at 0x4a5C685fEBfdBB4855EF0E0ac34580dbAD531a21.

https://mumbai.polygonscan.com/address/0x4a5C685fEBfdBB4855EF0E0ac34580dbAD531a21

Reply:
I deployed it myself before this one went up. I think it's going to be replaced anyway after we ship on-chain revocation.